x-pack/plugins/uptime/public/components/overview/alerts/alert_tls.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiExpression,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiLink,
+  EuiPopover,
+  EuiSpacer,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
+import { TlsTranslations } from './translations';
+
+interface SettingsMessageExpressionPopoverProps {
+  'aria-label': string;
+  description: string;
+  id: string;
+  setAlertFlyoutVisible: (value: boolean) => void;
+  value: string;
+}
+
+const SettingsMessageExpressionPopover: React.FC<SettingsMessageExpressionPopoverProps> = ({
+  'aria-label': ariaLabel,
+  description,
+  setAlertFlyoutVisible,
+  value,
+  id,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const kibana = useKibana();
+  const href = kibana.services?.application?.getUrlForApp('uptime#/settings');
```

Comment:
This a nice utility, but using Link component from react-router, you can just use route string from constants and you will not need this.

Reply:
Thanks for pointing that out, like it much better!

Reply:
Addressed in https://github.com/elastic/kibana/pull/63913/commits/d693a2a4c98018bbb1876ac5b14fe5efc0825ea6.